pkg/tfgen/generate.go

```diff
@@ -734,7 +734,7 @@ func (g *generator) gatherDataSource(rawname string,
 		cust := info.Fields[arg]
 
 		// Remember detailed information for every input arg (we will use it below).
-		if input(args[arg]) {
+		if input(args[arg]) && !isComputedOverride(cust) {
```

Comment:
Probably better to have `input` make this decision in case there are other callers: presumably those callers will also need to know about `MarkAsComputedOnly`.

Reply:
You mean like change the input func to look like this?

```
func input(sch *schema.Schema, info *tfbridge.SchemaInfo) bool {
	return sch.Optional || sch.Required && !(info.MarkAsComputedOnly != nil && *info.MarkAsComputedOnly)
}
```

Reply:
Needs some parens, but yes:

```
func input(sch *schema.Schema, info *tfbridge.SchemaInfo) bool {
	return (sch.Optional || sch.Required) && !(info.MarkAsComputedOnly != nil && *info.MarkAsComputedOnly)
}
```